match/lib/match/encrypt.rb

```diff
@@ -93,6 +93,7 @@ def crypt(path: nil, password: nil, encrypt: true)
       success = system(command.join(' '))
 
       UI.crash!("Error decrypting '#{path}'") unless success
+      sleep 0.1 until File.exist?(tmpfile)
```

Comment:
Is this something that we could do "if not mac, then sleep"?
Also, is just 0.1 enough or should it be something insanely long, like .250?

Reply:
Hm, only executing it on non-Mac systems seems like a good idea. I encountered this while testing fastlane on WSL, so it probably is not happening on Mac itself.

> Also, is just 0.1 enough or should it be something insanely long, like .250?

This code should be repeated until the file exists. (Although I realize this might not be the best idea as well... how can I do this better? I know that it just takes some time for the file to be where it should be :/ )

Reply:
WHoops, didn't catch the `until`
Spinning is great for stuff like this, but maybe timeout after X seconds (is 2 seconds long enough?) and throw an error? 
-Josh

Reply:
I honestly have no idea how long it takes - I don't even understand _why_ this is happening in the first place.

Reply:
I was thinking something like just setting a variable `count = 0` and then looping + incrementing count up to `count == 20 || File.exist?(tmpfile)` (20 * .1  is 1 second)
So whatever happens first, either 20 loops of waiting 0.1 seconds, or if the file exists, the loop will break

Reply:
I think/hope I implemented just that.